docker/suite/run_suite.sh

```diff
@@ -6,8 +6,8 @@ set -eu -o pipefail
 # installer could be local .tar installer or s3:// or http(s) URL
 #
 
-if [ -f $INSTALLER_URL ] ; then
-	INSTALLER_FILE='/robotest/installer.tar'
+if [ -f $INSTALLER_URL ] || [ -n "${WAIT_FOR_INSTALLER:-}" ]; then
```

Comment:
When would you not want to wait for installer?

Reply:
i.e. when we're fetching a known release